modules/useQueries.js

```diff
@@ -41,7 +27,7 @@ const useQueries = (createHistory) =>
         return location
 
       if (location.query == null)
-        location.query = parseQueryString(location.search.substring(1))
+        location.query = parseQueryString(location.search)
```

Comment:
`query-string` handles leading `?`. This is breaking for legacy `qs` users, but I don't think we need to make that accommodation here.